Python preparing for ada (#123)
Ada-Developers-Academy/jump-start: preparing-for-ada/README.md

# Ada Developers Academy JumpStart Curriculum

## The Ada Application Process

> You can read more about the Ada application process [on our website](https://adadevelopersacademy.org/process/).  With the online application you will submit a resume and answer some questions.  In Phase 2, you will be confronted with a coding challenge in [Python](https://www.python.org/) to solve and an interview to explain their solution and examine ways to extend it.  Applicants who proceed to Phase 3 will participate in a video interview.

Reviewer: I think it makes sense to link them to interview process on our website rather than repeating those instructions here. Should we give a little mention about what Phase 3 will involve (technical challenge and behavioral questions?).

> You can read more about the Ada application process [on our website](https://adadevelopersacademy.org/process/).  With the online application you will submit a resume and answer some questions.  In Phase 2, you will be confronted with a coding challenge in [Python](https://www.python.org/) to solve and an interview to explain their solution and examine ways to extend it.  Applicants who proceed to Phase 3 will participate in a video interview which will focus on behavioral questions to guage your passion for Ada's mission and software development, aptitude for learning in our collaborative and inclusive environment, and grit and determination to succeed.

Reviewer 2: ```suggestion
You can read more about the Ada application process [on our website](https://adadevelopersacademy.org/process/).  With the online application you will submit a resume and answer some questions.  

If you proceed to Phase 2, you will be confronted with a coding challenge in [Python](https://www.python.org/) to solve and an interview to explain your solution and examine ways to extend it.  

If you proceed to Phase 3 you will participate in a video interview which will focus on behavioral questions to gauge your passion for Ada's mission and software development, aptitude for learning in our collaborative and inclusive environment, grit and determination to succeed.
```

## What We Are Looking For

At Ada we want students who:

*  Demonstrate interest in coding and time spent self-learning
*  Have commitments and/or involvement with community/volunteer activities
*  Demonstrate examples of leadership
*  Communicate a clear desire to join Ada that goes beyond receiving a technical education
*  Demonstrate the ability to learn something new in order to accomplish a given task
*  Evidence the grit and determination necessary to enter Tech
*  Demonstrate the ability to persist through setbacks and learn from mistakes
*  Can clearly communicate their thought process, assumptions, and conclusions 

## Preparing for Ada

These lessons will help teach you the art of problem solving.

| Order | Lesson                                         |
| :---- | :--------------------------------------------- |
| 00    | [Applying to Ada - Phase 2](./applying-to-ada) |

> | 01    | [Problem Solving](./problem-solving/)          |

Reviewer: I think the algorithm / problem solving piece is a nice warm up, though not 100% necessary. Might be interested to see what Dee or Bethany thinks of this.

Author: Will ask them.
